.github/workflows/daily-nightly-jobs.yml

```diff
@@ -2,6 +2,7 @@ name: Daily nightly jobs
 on:
   schedule:
     - cron: "0 0 * * *" # every day at midnight
+  workflow_dispatch: {}
```

Comment:
I think this will allow us to kick off the nightly jobs any time we want. This might be good for verifying that the changes here are actually working without waiting days.